app/services/chat_memory.py

```
@@ -47,7 +47,16 @@ def _coerce_uuid(value: Any, field_name: str) -> uuid.UUID:
         except (TypeError, ValueError) as exc:
-            raise ValueError(f"{field_name} must be a valid UUID") from exc
+            # Generate a new UUID for invalid session IDs to prevent crashes
+            if field_name == "session_id":
+                logger.warning(
+                    "Invalid session_id provided, generating new UUID",
+                    invalid_session_id=str(value),
+                    error=str(exc)
+                )
+                return uuid.uuid4()
+            else:
+                raise ValueError(f"{field_name} must be a valid UUID") from exc
```

Ruff (0.14.0)

Avoid specifying long messages outside the exception class

(TRY003)